### PR TITLE
web/test_basic_views --> web/test_transactions

### PR DESCRIPTION
**Pull request by TimDaub**
Gonna add a new endpoint and view soon with #601. Currently all tests in `test_basic_views.py` test `web/views/transactions.py`, so I thought it would make sense to just rename it, so that we can have the tests in separate files.

### CODE REVIEW

**Comment by ttmc** [comment]
`web/test_basic_views.py` includes a test for the "basic info endpoint":

```python
def test_api_endpoint_shows_basic_info(client):
    from ...
``` 

Maybe that and any similar tests should be moved into a new file?

**Comment by TimDaub** [comment]
> Maybe that and any similar tests should be moved into a new file?

Solved in 5c0616e

File: tests/web/test_info.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def test_api_endpoint_shows_basic_info(client):
+    from bigchaindb import version
```
Comment (libscott) [inline tests/web/test_info.py]: Why is this in the function and not at the top?

**Comment by ttmc** [comment]
screenshot from 2016-11-15 12 01 35

Shouldn't that test (`test_api_endpoint_shows_basic_info`) also be removed from the renamed tests file?

**Comment by TimDaub** [comment]
> Shouldn't that test (test_api_endpoint_shows_basic_info) also be removed from the renamed tests file?

I messed that up. Sorry.

**Comment by TimDaub** [comment]
> Shouldn't that test (test_api_endpoint_shows_basic_info) also be removed from the renamed tests file?

Should be fixed now. @ttmc

File: tests/web/test_info.py (hunk 1 above)
Comment (TimDaub) [inline tests/web/test_info.py]: Shouldn't it be OK to import it in the test in this case though, as you're actually using it to do an assertion?

Comment (libscott) [inline tests/web/test_info.py]: @diminator, that still doesn't explain the reasoning, in `test_transaction.py` the imports are done over and over again, but the modules that are imported are not re-initialized each time (I just tested this), so I'm not sure what the reasoning behind it is.

Comment (sbellem) [inline tests/web/test_info.py]: @libscott the practice of putting some imports under function scope is based on http://docs.pylonsproject.org/en/latest/community/testing.html

**Comment by sbellem** [comment]
:+1:

File: tests/web/test_info.py (hunk 1 above)
Comment (libscott) [inline tests/web/test_info.py]: Ok, wasn't aware of this practice.

**Pull request merged**